godot_project/editor/rendering/virtual_motor_renderer/asset/motor_material.gdshader

```diff
@@ -46,7 +46,7 @@ void vertex() {
 
 void fragment() {
 	// Selectable dimming
-	vec4 texture_color = texture(albedo_texture, UV) * color;
+	vec4 texture_color = texture(albedo_texture, UV) * albedo;
```

Comment:
This was a mistake made on https://github.com/MSEP-one/msep.one/pull/253/files#diff-42dae2a5111f9f14f455337b9495d2546efd12129ee80972dd7c26203e25af20L48